cmd/kibosh/main.go

```diff
@@ -26,12 +26,35 @@ import (
 	"github.com/cf-platform-eng/kibosh/pkg/helm"
 	"github.com/cf-platform-eng/kibosh/pkg/httphelpers"
 	"github.com/cf-platform-eng/kibosh/pkg/k8s"
-	"github.com/cf-platform-eng/kibosh/pkg/operator"
 	"github.com/cf-platform-eng/kibosh/pkg/repository"
 	"github.com/cloudfoundry-community/go-cfclient"
 	"github.com/pivotal-cf/brokerapi"
 )
 
+type ClusterFactory struct {
+	ClusterCredentials config.ClusterCredentials
+}
+
+func (cf ClusterFactory) DefaultCluster() (k8s.Cluster, error) {
+	return k8s.NewCluster(&cf.ClusterCredentials)
+}
+
+type HelmClientFactory struct {
```

Comment:
I think this belongs in the `helm` package, not main. Main's role is to be really small and not defining anything new, just wire up things with their real implementations.